Unit test failure in vicadmin

Was this build before @cgtexmex restarted the build:
https://ci.vmware.run/vmware/vic/4290
```
time="2016-08-23T17:59:19Z" level=warning msg="Unable to connect: SDK URL () could not be parsed: %!s(<nil>)" 
time="2016-08-23T17:59:19Z" level=debug msg="[ END ] [github.com/vmware/vic/cmd/vicadmin.client] [12.52µs] " 
time="2016-08-23T17:59:19Z" level=debug msg="[ END ] [github.com/vmware/vic/cmd/vicadmin.findDatastore] [34.926µs] " 
time="2016-08-23T17:59:19Z" level=warning msg="SDK URL () could not be parsed: %!s(<nil>)" 
time="2016-08-23T17:59:19Z" level=debug msg="[ END ] [github.com/vmware/vic/cmd/vicadmin.configureReaders] [181.409µs] " 
2016/08/23 17:59:19 Testing TestLogTail
2016/08/23 17:59:19 GET https://localhost:36946/logs/tail/vicadmin.log:
time="2016-08-23T17:59:19Z" level=debug msg="[BEGIN] [github.com/vmware/vic/cmd/vicadmin.(*server).tailFiles]" 
time="2016-08-23T17:59:19Z" level=debug msg="[BEGIN] [github.com/vmware/vic/cmd/vicadmin.tailFile] ./vicadmin.log" 
time="2016-08-23T17:59:19Z" level=debug msg="[BEGIN] [github.com/vmware/vic/cmd/vicadmin.findSeekPos]" 
time="2016-08-23T17:59:19Z" level=debug msg="[ END ] [github.com/vmware/vic/cmd/vicadmin.findSeekPos] [8.665µs] " 
2016/08/23 17:59:19 Seeked ./vicadmin.log - &{Offset:0 Whence:0}
SIGQUIT: quit
PC=0x4a4a1a m=0

goroutine 95 [syscall, 9 minutes]:
syscall.Syscall6(0xe8, 0x13, 0xc4200efc14, 0x7, 0xffffffffffffffff, 0x0, 0x0, 0x0, 0x101020203030303, 0x0)
	/usr/local/go/src/syscall/asm_linux_amd64.s:44 +0x5 fp=0xc4200efb40 sp=0xc4200efb38
syscall.EpollWait(0x13, 0xc4200efc14, 0x7, 0x7, 0xffffffffffffffff, 0x0, 0x0, 0x0)
	/usr/local/go/src/syscall/zsyscall_linux_amd64.go:365 +0x77 fp=0xc4200efbb0 sp=0xc4200efb40
github.com/vmware/vic/vendor/github.com/hpcloud/tail/vendor/gopkg.in/fsnotify%2ev1.(*fdPoller).wait(0xc420186740, 0xe34800, 0xc420186740, 0x0)
	/drone/src/github.com/vmware/vic/vendor/github.com/hpcloud/tail/vendor/gopkg.in/fsnotify.v1/inotify_poller.go:85 +0x91 fp=0xc4200efca0 sp=0xc4200efbb0
github.com/vmware/vic/vendor/github.com/hpcloud/tail/vendor/gopkg.in/fsnotify%2ev1.(*Watcher).readEvents(0xc42001bdb0)
	/drone/src/github.com/vmware/vic/vendor/github.com/hpcloud/tail/vendor/gopkg.in/fsnotify.v1/inotify.go:179 +0x196 fp=0xc4200fff58 sp=0xc4200efca0
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:2086 +0x1 fp=0xc4200fff60 sp=0xc4200fff58
created by github.com/vmware/vic/vendor/github.com/hpcloud/tail/vendor/gopkg.in/fsnotify%2ev1.NewWatcher
	/drone/src/github.com/vmware/vic/vendor/github.com/hpcloud/tail/vendor/gopkg.in/fsnotify.v1/inotify.go:58 +0x2cc

goroutine 1 [chan receive, 9 minutes]:
testing.(*T).Run(0xc4200783c0, 0xdcd489, 0xb, 0xe34708, 0xc4201dbc01)
	/usr/local/go/src/testing/testing.go:647 +0x316
testing.RunTests.func1(0xc4200783c0)
	/usr/local/go/src/testing/testing.go:793 +0x6d
testing.tRunner(0xc4200783c0, 0xc420037d98)
	/usr/local/go/src/testing/testing.go:610 +0x81
testing.RunTests(0xe34a88, 0x1603420, 0x5, 0x5, 0x4114ce)
	/usr/local/go/src/testing/testing.go:799 +0x2f5
testing.(*M).Run(0xc420037ef8, 0xe01093)
	/usr/local/go/src/testing/testing.go:743 +0x85
main.main()
	github.com/vmware/vic/cmd/vicadmin/_test/_testmain.go:112 +0x1bd

goroutine 17 [syscall, 9 minutes, locked to thread]:
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:2086 +0x1

goroutine 19 [select, 9 minutes]:
net/http.(*persistConn).writeLoop(0xc42000af00)
	/usr/local/go/src/net/http/transport.go:1641 +0x3bd
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:1058 +0x50e

goroutine 6 [syscall, 9 minutes]:
os/signal.signal_recv(0x0)
	/usr/local/go/src/runtime/sigqueue.go:116 +0x157
os/signal.loop()
	/usr/local/go/src/os/signal/signal_unix.go:22 +0x22
created by os/signal.init.1
	/usr/local/go/src/os/signal/signal_unix.go:28 +0x41

goroutine 7 [IO wait, 9 minutes]:
net.runtime_pollWait(0x7f526c8f9f00, 0x72, 0x0)
	/usr/local/go/src/runtime/netpoll.go:160 +0x59
net.(*pollDesc).wait(0xc4201ee060, 0x72, 0xc420034cf8, 0xc4200140b8)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x38
net.(*pollDesc).waitRead(0xc4201ee060, 0x16105c0, 0xc4200140b8)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x34
net.(*netFD).accept(0xc4201ee000, 0x0, 0x160ec40, 0xc4201e8380)
	/usr/local/go/src/net/fd_unix.go:419 +0x238
net.(*TCPListener).accept(0xc4201f2000, 0x29e8d60800, 0x0, 0x0)
	/usr/local/go/src/net/tcpsock_posix.go:132 +0x2e
net.(*TCPListener).AcceptTCP(0xc4201f2000, 0xc420034e20, 0xc420034e28, 0xc420034e18)
	/usr/local/go/src/net/tcpsock.go:209 +0x49
net/http.tcpKeepAliveListener.Accept(0xc4201f2000, 0xe34c98, 0xc4201e6100, 0x1613f80, 0xc4201ec0c0)
	/usr/local/go/src/net/http/server.go:2608 +0x2f
net/http.(*Server).Serve(0xc4201e6000, 0x1613840, 0xc4201f2000, 0x0, 0x0)
	/usr/local/go/src/net/http/server.go:2273 +0x1ce
net/http.(*Server).ListenAndServe(0xc4201e6000, 0xc4201e6000, 0x0)
	/usr/local/go/src/net/http/server.go:2219 +0xb4
net/http.ListenAndServe(0xc4201e4007, 0xe, 0x0, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/http/server.go:2351 +0xa0
github.com/vmware/vic/lib/pprof.StartPprof.func1(0xc4201e4007, 0xe)
	/drone/src/github.com/vmware/vic/lib/pprof/pprof.go:91 +0x4b
created by github.com/vmware/vic/lib/pprof.StartPprof
	/drone/src/github.com/vmware/vic/lib/pprof/pprof.go:92 +0x25a

goroutine 8 [IO wait, 9 minutes]:
net.runtime_pollWait(0x7f526c8f9cc0, 0x72, 0x0)
	/usr/local/go/src/runtime/netpoll.go:160 +0x59
net.(*pollDesc).wait(0xc4201d3fe0, 0x72, 0xc42003ac88, 0xc4200140b8)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x38
net.(*pollDesc).waitRead(0xc4201d3fe0, 0x16105c0, 0xc4200140b8)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x34
net.(*netFD).accept(0xc4201d3f80, 0x0, 0x160ec40, 0xc42068aa00)
	/usr/local/go/src/net/fd_unix.go:419 +0x238
net.(*TCPListener).accept(0xc420028778, 0x29e8d60800, 0x0, 0x0)
	/usr/local/go/src/net/tcpsock_posix.go:132 +0x2e
net.(*TCPListener).AcceptTCP(0xc420028778, 0xc42003adb0, 0xc42003adb8, 0xc42003ada8)
	/usr/local/go/src/net/tcpsock.go:209 +0x49
net/http.tcpKeepAliveListener.Accept(0xc420028778, 0xe34c98, 0xc420346f80, 0x1613f80, 0xc4201fda10)
	/usr/local/go/src/net/http/server.go:2608 +0x2f
net/http.(*Server).Serve(0xc42007ed00, 0x1613840, 0xc420028778, 0x0, 0x0)
	/usr/local/go/src/net/http/server.go:2273 +0x1ce
net/http.(*Server).ListenAndServe(0xc42007ed00, 0xc42007ed00, 0x0)
	/usr/local/go/src/net/http/server.go:2219 +0xb4
net/http.ListenAndServe(0xdcfad7, 0xe, 0x0, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/http/server.go:2351 +0xa0
github.com/vmware/vic/cmd/vicadmin.init.2.func1(0xdc70b3, 0x4, 0x0, 0x0, 0x0, 0xdcfad7, 0xe, 0x0, 0x0, 0x0, ...)
	/drone/src/github.com/vmware/vic/cmd/vicadmin/vicadm_test.go:58 +0x4e
created by github.com/vmware/vic/cmd/vicadmin.init.2
	/drone/src/github.com/vmware/vic/cmd/vicadmin/vicadm_test.go:59 +0xde

goroutine 86 [IO wait, 9 minutes]:
net.runtime_pollWait(0x7f526c8f9780, 0x72, 0xe)
	/usr/local/go/src/runtime/netpoll.go:160 +0x59
net.(*pollDesc).wait(0xc4201d2450, 0x72, 0xc420363468, 0xc4200140b8)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x38
net.(*pollDesc).waitRead(0xc4201d2450, 0x16105c0, 0xc4200140b8)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x34
net.(*netFD).Read(0xc4201d23f0, 0xc4202ecc00, 0x400, 0x400, 0x0, 0x16105c0, 0xc4200140b8)
	/usr/local/go/src/net/fd_unix.go:243 +0x1a1
net.(*conn).Read(0xc4201f20c0, 0xc4202ecc00, 0x400, 0x400, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/net.go:173 +0x70
crypto/tls.(*block).readFromUntil(0xc420233410, 0x7f526c941308, 0xc4201f20c0, 0x5, 0xc4201f20c0, 0x0)
	/usr/local/go/src/crypto/tls/conn.go:472 +0x91
crypto/tls.(*Conn).readRecord(0xc4202ee380, 0xe35417, 0xc4202ee480, 0x0)
	/usr/local/go/src/crypto/tls/conn.go:574 +0xc4
crypto/tls.(*Conn).Read(0xc4202ee380, 0xc420194000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	/usr/local/go/src/crypto/tls/conn.go:1109 +0x116
net/http.(*persistConn).Read(0xc420310600, 0xc420194000, 0x1000, 0x1000, 0x14, 0x0, 0x0)
	/usr/local/go/src/net/http/transport.go:1256 +0x154
bufio.(*Reader).fill(0xc420018420)
	/usr/local/go/src/bufio/bufio.go:97 +0x10c
bufio.(*Reader).ReadSlice(0xc420018420, 0xc42023380a, 0x2, 0x2, 0x2, 0x0, 0x0)
	/usr/local/go/src/bufio/bufio.go:330 +0xb5
net/http/internal.readChunkLine(0xc420018420, 0xc420018420, 0xc4202338b0, 0x2, 0x2, 0x2)
	/usr/local/go/src/net/http/internal/chunked.go:110 +0x34
net/http/internal.(*chunkedReader).beginChunk(0xc420233890)
	/usr/local/go/src/net/http/internal/chunked.go:47 +0x32
net/http/internal.(*chunkedReader).Read(0xc420233890, 0xc4201a0000, 0xdd, 0x2000, 0x4565d0, 0xc420363aa0, 0xc420363aa8)
	/usr/local/go/src/net/http/internal/chunked.go:77 +0x7b
net/http.(*body).readLocked(0xc4201f1980, 0xc4201a0000, 0xdd, 0x2000, 0xc42022d9a0, 0x0, 0x0)
	/usr/local/go/src/net/http/transfer.go:651 +0x61
net/http.(*body).Read(0xc4201f1980, 0xc4201a0000, 0xdd, 0x2000, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/http/transfer.go:643 +0xf6
net/http.(*bodyEOFSignal).Read(0xc4201f19c0, 0xc4201a0000, 0xdd, 0x2000, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/http/transport.go:1969 +0x116
io.(*LimitedReader).Read(0xc420192de0, 0xc4201a0000, 0x2000, 0x2000, 0xf, 0x0, 0x0)
	/usr/local/go/src/io/io.go:436 +0x6c
io/ioutil.devNull.ReadFrom(0x0, 0x160df80, 0xc420192de0, 0xc2fc40, 0xe31c01, 0x7f526c8fe728)
	/usr/local/go/src/io/ioutil/ioutil.go:151 +0x85
io/ioutil.(*devNull).ReadFrom(0xc420014560, 0x160df80, 0xc420192de0, 0xc420363ca0, 0x1, 0x18)
	<autogenerated>:9 +0x66
io.copyBuffer(0x160f980, 0xc420014560, 0x160df80, 0xc420192de0, 0x0, 0x0, 0x0, 0xce6a80, 0x160e501, 0xc420192de0)
	/usr/local/go/src/io/io.go:384 +0x323
io.Copy(0x160f980, 0xc420014560, 0x160df80, 0xc420192de0, 0xc2cd20, 0xc56d00, 0xc420363d00)
	/usr/local/go/src/io/io.go:360 +0x68
io.CopyN(0x160f980, 0xc420014560, 0x160e580, 0xc4201f19c0, 0x100, 0x0, 0xf, 0x1)
	/usr/local/go/src/io/io.go:336 +0x8c
github.com/vmware/vic/cmd/vicadmin.TestLogTail(0xc420078180)
	/drone/src/github.com/vmware/vic/cmd/vicadmin/vicadm_test.go:281 +0x6a1
testing.tRunner(0xc420078180, 0xe34708)
	/usr/local/go/src/testing/testing.go:610 +0x81
created by testing.(*T).Run
	/usr/local/go/src/testing/testing.go:646 +0x2ec

goroutine 109 [chan receive, 9 minutes]:
github.com/vmware/vic/cmd/vicadmin.(*server).tailFiles(0xc42000d7c0, 0x1613380, 0xc4201f8ea0, 0xc4202f4d20, 0xc42033ac68, 0x1, 0x1)
	github.com/vmware/vic/cmd/vicadmin/_test/_obj_test/server.go:318 +0x311
github.com/vmware/vic/cmd/vicadmin.(*server).serve.func2(0x1613380, 0xc4201f8ea0, 0xc4202f4d20)
	github.com/vmware/vic/cmd/vicadmin/_test/_obj_test/server.go:170 +0x9a
net/http.HandlerFunc.ServeHTTP(0xc4202251a0, 0x1613380, 0xc4201f8ea0, 0xc4202f4d20)
	/usr/local/go/src/net/http/server.go:1726 +0x44
net/http.(*ServeMux).ServeHTTP(0xc420369f80, 0x1613380, 0xc4201f8ea0, 0xc4202f4d20)
	/usr/local/go/src/net/http/server.go:2022 +0x7f
net/http.serverHandler.ServeHTTP(0xc4201e6c80, 0x1613380, 0xc4201f8ea0, 0xc4202f4d20)
	/usr/local/go/src/net/http/server.go:2202 +0x7d
net/http.(*conn).serve(0xc4201e6f80, 0x1613ec0, 0xc420650540)
	/usr/local/go/src/net/http/server.go:1579 +0x4b7
created by net/http.(*Server).Serve
	/usr/local/go/src/net/http/server.go:2293 +0x44d

goroutine 18 [IO wait, 9 minutes]:
net.runtime_pollWait(0x7f526c8f9d80, 0x72, 0x6)
	/usr/local/go/src/runtime/netpoll.go:160 +0x59
net.(*pollDesc).wait(0xc4201d3f70, 0x72, 0xc4200396f0, 0xc4200140b8)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x38
net.(*pollDesc).waitRead(0xc4201d3f70, 0x16105c0, 0xc4200140b8)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x34
net.(*netFD).Read(0xc4201d3f10, 0xc4202eb800, 0x800, 0x800, 0x0, 0x16105c0, 0xc4200140b8)
	/usr/local/go/src/net/fd_unix.go:243 +0x1a1
net.(*conn).Read(0xc420028770, 0xc4202eb800, 0x800, 0x800, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/net.go:173 +0x70
crypto/tls.(*block).readFromUntil(0xc4201fd920, 0x7f526c941308, 0xc420028770, 0x5, 0xc420028770, 0x28)
	/usr/local/go/src/crypto/tls/conn.go:472 +0x91
crypto/tls.(*Conn).readRecord(0xc420212380, 0xe35417, 0xc420212480, 0xc4200756c0)
	/usr/local/go/src/crypto/tls/conn.go:574 +0xc4
crypto/tls.(*Conn).Read(0xc420212380, 0xc42031c000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	/usr/local/go/src/crypto/tls/conn.go:1109 +0x116
net/http.(*persistConn).Read(0xc42000af00, 0xc42031c000, 0x1000, 0x1000, 0x66ea60, 0xc420039b58, 0x405a4d)
	/usr/local/go/src/net/http/transport.go:1256 +0x154
bufio.(*Reader).fill(0xc4203085a0)
	/usr/local/go/src/bufio/bufio.go:97 +0x10c
bufio.(*Reader).Peek(0xc4203085a0, 0x1, 0xc420039bbd, 0x1, 0x0, 0xc420308600, 0x0)
	/usr/local/go/src/bufio/bufio.go:129 +0x62
net/http.(*persistConn).readLoop(0xc42000af00)
	/usr/local/go/src/net/http/transport.go:1413 +0x1a1
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:1057 +0x4e9

goroutine 16 [IO wait, 9 minutes]:
net.runtime_pollWait(0x7f526c8f9c00, 0x72, 0x8)
	/usr/local/go/src/runtime/netpoll.go:160 +0x59
net.(*pollDesc).wait(0xc4202345a0, 0x72, 0xc4200454c0, 0xc4200140b8)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x38
net.(*pollDesc).waitRead(0xc4202345a0, 0x16105c0, 0xc4200140b8)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x34
net.(*netFD).Read(0xc420234540, 0xc4201e1c00, 0x400, 0x400, 0x0, 0x16105c0, 0xc4200140b8)
	/usr/local/go/src/net/fd_unix.go:243 +0x1a1
net.(*conn).Read(0xc420028780, 0xc4201e1c00, 0x400, 0x400, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/net.go:173 +0x70
crypto/tls.(*block).readFromUntil(0xc4202326c0, 0x7f526c941308, 0xc420028780, 0x5, 0xc420028780, 0xc4200455e8)
	/usr/local/go/src/crypto/tls/conn.go:472 +0x91
crypto/tls.(*Conn).readRecord(0xc420212700, 0xe35417, 0xc420212800, 0x430a00)
	/usr/local/go/src/crypto/tls/conn.go:574 +0xc4
crypto/tls.(*Conn).Read(0xc420212700, 0xc42031e000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	/usr/local/go/src/crypto/tls/conn.go:1109 +0x116
net/http.(*connReader).Read(0xc4202e0d40, 0xc42031e000, 0x1000, 0x1000, 0xc420045930, 0x4a49e4, 0x5dd777)
	/usr/local/go/src/net/http/server.go:586 +0x144
bufio.(*Reader).fill(0xc420019bc0)
	/usr/local/go/src/bufio/bufio.go:97 +0x10c
bufio.(*Reader).ReadSlice(0xc420019bc0, 0xa, 0x0, 0x1e, 0xc41ffcc053, 0x33, 0x0)
	/usr/local/go/src/bufio/bufio.go:330 +0xb5
bufio.(*Reader).ReadLine(0xc420019bc0, 0xc4202f43c0, 0xf0, 0xf0, 0xdb3ee0, 0xc420236c00, 0x7f526c9994b0)
	/usr/local/go/src/bufio/bufio.go:359 +0x37
net/textproto.(*Reader).readLineSlice(0xc42067f560, 0xc420045a98, 0xc420045a98, 0x411cf8, 0xf0, 0xdb3ee0)
	/usr/local/go/src/net/textproto/reader.go:55 +0x5e
net/textproto.(*Reader).ReadLine(0xc42067f560, 0xc4202f43c0, 0x4566d0, 0xc4201f0900, 0xc420045b68)
	/usr/local/go/src/net/textproto/reader.go:36 +0x2f
net/http.readRequest(0xc420019bc0, 0x0, 0xc4202f43c0, 0x0, 0x0)
	/usr/local/go/src/net/http/request.go:793 +0xa5
net/http.(*conn).readRequest(0xc42007f100, 0x1613ec0, 0xc42022f280, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/http/server.go:765 +0x10d
net/http.(*conn).serve(0xc42007f100, 0x1613ec0, 0xc42022f280)
	/usr/local/go/src/net/http/server.go:1532 +0x3d3
created by net/http.(*Server).Serve
	/usr/local/go/src/net/http/server.go:2293 +0x44d

goroutine 87 [IO wait, 9 minutes]:
net.runtime_pollWait(0x7f526c8f9e40, 0x72, 0x0)
	/usr/local/go/src/runtime/netpoll.go:160 +0x59
net.(*pollDesc).wait(0xc420235f70, 0x72, 0xc42004bc48, 0xc4200140b8)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x38
net.(*pollDesc).waitRead(0xc420235f70, 0x16105c0, 0xc4200140b8)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x34
net.(*netFD).accept(0xc420235f10, 0x0, 0x160ec40, 0xc420225ba0)
	/usr/local/go/src/net/fd_unix.go:419 +0x238
net.(*TCPListener).accept(0xc4201f2088, 0xc420028148, 0xc4201e0800, 0x7f526c999000)
	/usr/local/go/src/net/tcpsock_posix.go:132 +0x2e
net.(*TCPListener).Accept(0xc4201f2088, 0x435aee, 0xc42004bd50, 0x657e1d, 0x456f40)
	/usr/local/go/src/net/tcpsock.go:222 +0x49
crypto/tls.(*listener).Accept(0xc42068b860, 0xe34c98, 0xc4201e6f80, 0x1613f80, 0xc420233320)
	/usr/local/go/src/crypto/tls/tls.go:52 +0x37
net/http.(*Server).Serve(0xc4201e6c80, 0x1613080, 0xc42068b860, 0x0, 0x0)
	/usr/local/go/src/net/http/server.go:2273 +0x1ce
github.com/vmware/vic/cmd/vicadmin.(*server).serve(0xc42000d7c0, 0x0, 0x0)
	github.com/vmware/vic/cmd/vicadmin/_test/_obj_test/server.go:182 +0x916
created by github.com/vmware/vic/cmd/vicadmin.TestLogTail
	/drone/src/github.com/vmware/vic/cmd/vicadmin/vicadm_test.go:242 +0x23a

goroutine 93 [select, 9 minutes]:
github.com/vmware/vic/vendor/github.com/hpcloud/tail.(*Tail).waitForChanges(0xc4201a4dc0, 0xc420014160, 0x160ac40)
	/drone/src/github.com/vmware/vic/vendor/github.com/hpcloud/tail/tail.go:350 +0xe5a
github.com/vmware/vic/vendor/github.com/hpcloud/tail.(*Tail).tailFileSync(0xc4201a4dc0)
	/drone/src/github.com/vmware/vic/vendor/github.com/hpcloud/tail/tail.go:311 +0x313
created by github.com/vmware/vic/vendor/github.com/hpcloud/tail.TailFile
	/drone/src/github.com/vmware/vic/vendor/github.com/hpcloud/tail/tail.go:133 +0x1f8

goroutine 94 [select, 9 minutes]:
github.com/vmware/vic/vendor/github.com/hpcloud/tail/watch.(*InotifyTracker).run(0xc420650840)
	/drone/src/github.com/vmware/vic/vendor/github.com/hpcloud/tail/watch/inotify_tracker.go:236 +0x45b
created by github.com/vmware/vic/vendor/github.com/hpcloud/tail/watch.glob..func1
	/drone/src/github.com/vmware/vic/vendor/github.com/hpcloud/tail/watch/inotify_tracker.go:54 +0x233

goroutine 91 [IO wait, 9 minutes]:
net.runtime_pollWait(0x7f526c8f9840, 0x72, 0xf)
	/usr/local/go/src/runtime/netpoll.go:160 +0x59
net.(*pollDesc).wait(0xc420017480, 0x72, 0xc420674398, 0xc4200140b8)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x38
net.(*pollDesc).waitRead(0xc420017480, 0x16105c0, 0xc4200140b8)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x34
net.(*netFD).Read(0xc420017420, 0xc42002bc00, 0x400, 0x400, 0x0, 0x16105c0, 0xc4200140b8)
	/usr/local/go/src/net/fd_unix.go:243 +0x1a1
net.(*conn).Read(0xc420028148, 0xc42002bc00, 0x400, 0x400, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/net.go:173 +0x70
crypto/tls.(*block).readFromUntil(0xc4202333e0, 0x7f526c941308, 0xc420028148, 0x5, 0xc420028148, 0x0)
	/usr/local/go/src/crypto/tls/conn.go:472 +0x91
crypto/tls.(*Conn).readRecord(0xc420212a80, 0xe35417, 0xc420212b80, 0x411cf8)
	/usr/local/go/src/crypto/tls/conn.go:574 +0xc4
crypto/tls.(*Conn).Read(0xc420212a80, 0xc4201fe9ac, 0x1, 0x1, 0x0, 0x0, 0x0)
	/usr/local/go/src/crypto/tls/conn.go:1109 +0x116
net/http.(*connReader).closeNotifyAwaitActivityRead(0xc420186440, 0xc420308c00, 0xc4201864e0)
	/usr/local/go/src/net/http/server.go:602 +0x71
created by net/http.(*connReader).startBackgroundRead
	/usr/local/go/src/net/http/server.go:597 +0x98

goroutine 96 [select, 9 minutes]:
github.com/vmware/vic/vendor/github.com/hpcloud/tail/watch.(*InotifyFileWatcher).ChangeEvents.func1(0xc420186620, 0xc4201a4e38, 0xc420186760)
	/drone/src/github.com/vmware/vic/vendor/github.com/hpcloud/tail/watch/inotify.go:88 +0x408
created by github.com/vmware/vic/vendor/github.com/hpcloud/tail/watch.(*InotifyFileWatcher).ChangeEvents
	/drone/src/github.com/vmware/vic/vendor/github.com/hpcloud/tail/watch/inotify.go:125 +0x155

goroutine 110 [select, 9 minutes]:
net/http.(*persistConn).readLoop(0xc420310600)
	/usr/local/go/src/net/http/transport.go:1536 +0x9c9
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:1057 +0x4e9

goroutine 92 [select, 9 minutes]:
github.com/vmware/vic/cmd/vicadmin.tailFile(0x160ad00, 0xc420186520, 0xc420322c00, 0xe, 0xc4201f20c8, 0x0, 0x0)
	github.com/vmware/vic/cmd/vicadmin/_test/_obj_test/logs.go:392 +0x4e7
created by github.com/vmware/vic/cmd/vicadmin.(*server).tailFiles
	github.com/vmware/vic/cmd/vicadmin/_test/_obj_test/server.go:314 +0x2c4

goroutine 111 [select, 9 minutes]:
net/http.(*persistConn).writeLoop(0xc420310600)
	/usr/local/go/src/net/http/transport.go:1641 +0x3bd
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:1058 +0x50e

rax    0xfffffffffffffffc
rbx    0xc42001c00c
rcx    0x4a4a1a
rdx    0x7
rdi    0x13
rsi    0xc4200efc14
rbp    0xc4200efba0
rsp    0xc4200efb38
r8     0x0
r9     0x0
r10    0xffffffffffffffff
r11    0x206
r12    0x0
r13    0xf3
r14    0x8
r15    0x3
rip    0x4a4a1a
rflags 0x206
cs     0x33
fs     0x0
gs     0x0
*** Test killed with quit: ran too long (10m0s).
FAIL	github.com/vmware/vic/cmd/vicadmin	600.008s
Makefile:227: recipe for target 'test-job-github.com/vmware/vic/cmd' failed
[info] build failed (exit code 2)
```

--- a/cmd/vicadmin/vicadm_test.go
+++ b/cmd/vicadmin/vicadm_test.go
@@ -251,15 +251,19 @@ func TestLogTail(t *testing.T) {
 		"/logs/tail/" + name,
 	}
 
-	i := 0
-
 	u := url.URL{
 		// User:   url.UserPassword("root", "thisisinsecure"),
 		Scheme: "https",
 		Host:   fmt.Sprintf("localhost:%d", port),
 	}
 
-	f.WriteString("this is line 0\n")
+	str := "The quick brown fox jumps over the lazy dog.\n"
+
+	// Pre-populate the log file
+	for i := 0; i < tailLines; i++ {
+		f.WriteString(str)
+	}
+
 	log.Printf("Testing TestLogTail\n")
 	for _, path := range paths {
 		u.Path = path
@@ -270,15 +274,18 @@ func TestLogTail(t *testing.T) {
 			t.Fatal(err)
 		}
 
+		// Each line written to the log file has enough bytes to ensure
+		// that 8 lines make up at least 256 bytes. This is in case this
+		// goroutine finishes writing all lines before tail kicks in.
 		go func() {
 			for j := 1; j < 512; j++ {
-				i++
-				f.WriteString(fmt.Sprintf("this is line %d\n", i))
+				f.WriteString(str)
 			}
 		}()
 
 		size := int64(256)
-		n, _ := io.CopyN(out, res.Body, size)
+		n, err := io.CopyN(out, res.Body, size)
+		assert.NoError(t, err)
 		out.Write([]byte("...\n"))
 
 		assert.Equal(t, size, n)

